README.md

```diff
@@ -0,0 +1,203 @@
+# DNS based service discovery plugin for [Fluentd](https://fluentd.org)
+
+The `dns` service discovery plugin updates target services by DNS list.
+
+This plugin uses `Socket.getaddrinfo` internally.
+
+## Requirements
+
+- Fluentd v1 or later
```

Comment:
Service discovery is supported in Fluentd v1.8.0 or later.
Should we point in accurate release version?